index.js

```diff
@@ -37,7 +37,7 @@ Datahub.prototype._crud = function (url, method, data) {
   }
 
   if (method === 'GET') {
-    options.json = true;
+    options.json = (options.json == null) ? true : options.json;
```

Comment:
options.json = (!options.json) ? true : options.json;

Will this work for you?